### ISSUE TITLE
Add option to display the URL as a QR code

### ISSUE THREAD

**Issue by larryw3i** [lwsjs/local-web-server#148]
Maybe generating qrcode is been needed, it's convenient if get my laptop files just scan the qrcode 

![image](https://user-images.githubusercontent.com/24457116/81257078-f339c400-9064-11ea-8507-a85a3796aaf5.png)

**Comment by tiagostutz** [comment]
Interesting... for development porpuses this could be really useful.
This could be one of the "View options", right?

**Comment by 75lb** [comment]
yes, `ws --qr` would be a useful feature. 👍 

Which URL should the QR encode? When you run `ws`, three URLs are printed: the hostname, local IP and one or more network interface IP addresses.  

```
$ ws
Listening on http://mba4.local:8000, http://127.0.0.1:8000, http://192.168.0.200:8000
```

Which of these URLs should the QR represent? We need to pass in an argument which identifies which URL to display as QR. 

```
$ ws --qr en0
<print QR code for network interface with name 'en0'>
```

Please implement in [lws](https://github.com/lwsjs/lws). You'll need update the built-in cli-view.

1. Clone the latest lws code.
1. Add a `list-network-interfaces` option [here](https://github.com/lwsjs/lws/blob/master/lib/view/cli-view.js#L16). Make it a boolean flag. OptionDefinition objects are documented [here](https://github.com/75lb/command-line-args/blob/master/doc/option-definition.md).
1. At [this point](https://github.com/lwsjs/lws/blob/master/lib/view/cli-view.js#L50), if `config.listNetworkInterfaces` is set, print `value` (the address list) to the console so the user can see the network interface names.
1. Next, add a `qr` option definition [here](https://github.com/lwsjs/lws/blob/master/lib/view/cli-view.js#L16) with type `String`. The value passed to `--vr` should be an interface name. 
2. At [this point](https://github.com/lwsjs/lws/blob/master/lib/view/cli-view.js#L50), check if `config.vr` is set. If so, search `value` (an array of objects) for the object with a `name` matching the supplied `config.vr` value. Print a QR code for this object's `url` value.

**Comment by tiagostutz** [comment]
Nice! I'm going to work on this. 

If the user does not pass any argument we could print by default the URL that matches one of the (IETF private address)[https://tools.ietf.org/html/rfc1918#page-4] because that's probably the address the user is looking for when scanning with an external device that is in the same private (wi-fi) network. Right?

**Comment by 75lb** [comment]
Yes, that sounds like a sensible default - using a mobile device on the same LAN is a common use case. 

Would that default also work when both your laptop and mobile device are connected to a mobile network? Would be a bonus if so.

**Comment by tiagostutz** [comment]
Like the laptop using the mobile as a hotspot, right? 
If so, it would work.

**Comment by tiagostutz** [comment]
@larryw3i this issue is being built and discussed here https://github.com/lwsjs/lws/issues/13

**Comment by 75lb** [comment]
Released in [v4.1.0](https://github.com/lwsjs/local-web-server/releases/tag/v4.1.0). Cheers! 👍
